source/style.rst

```diff
@@ -21,30 +24,35 @@ Code
 ====
 Our code follows `PEP-8`_ guidelines. Please look there for any code style
 matters. Since this is a very widely used standard, Google is also very helpful
-if you have any questions.
+if you have any questions. To standardize our convention for strings, please
+use single quotes except for when the string itself contains single quotes.
 
 Docstrings
 ==========
 Please write docstrings for all modules, classes, and functions/methods. In
 general, our docstrings should mostly follow `PEP-257`_ with the exception
 of an allowance for 2-line docstrings.
 
-General Rules:
---------------
+General Rules
+-------------
 * Always use ``"""triple double quotes"""`` for docstrings.
 * End all summaries/descriptions with periods.
 * Built-in types (like str and int) should not be quoted or backticked.
 * Numeric values should be left unbackticked (e.g. 1).
 * Strings should be quoted (e.g. 'foo').
-* Use the ``:keyword:`` directive as appropriate (e.g. \:keyword:\`True\`).
+* Built-in constants and keywords can be emphasized with backticks. An
+  appropriate reference to python docs with be automatically generated
+  (e.g. \`True\` or \`assert\`).
+* args and kwargs should be noted without asterisks (e.g. kwargs not
```

Comment:
Where'd you find this one? I've at least recently used `*args` and `**kwargs` in docstrings as it means something significantly different from `args` and `kwargs`.

Reply:
Yeah, I was a little hesistant to do it. I was basing it off of tk0miya's preference as referenced in https://github.com/sphinx-doc/sphinx/issues/2869. The other option is escaping.

Reply:
I think there has to be a good balance between programmer readability (which numpy-formatted docstrings are fantastic for) and sphinx/rst parse-ability. In my opinion, this goes a bit too far towards the latter.

I wonder, is there a docstring preprocess hook we could use to replace `**kwargs` with `\**kwargs` or something?

If others feel I'm making a mountain out of a mole hill here, feel free to jump in ;)

Reply:
No, it's definitely something that should be explored further.
I'll look into some sort of fix but so far it doesn't look like it'll be simple.

Reply:
Ok, so what seems to work is:
- If you want to mention **kwargs in a docstring, refer to them as ` ``**kwargs`` `
- If you want to list them in the parameters section, the backticks aren't necessary

Reply:
OK, that's great news. I have no complaints about that, then.